aim/db/migration/alembic_migrations/versions/fabed2911290_modify_host_column_size.py

```diff
@@ -29,20 +29,46 @@
 
 from alembic import op
 import sqlalchemy as sa
+from sqlalchemy.sql.sqltypes import VARCHAR
 
 
+_INSPECTOR = None
+
 TABLES = ('aim_endpoint_group_static_paths',
           'aim_concrete_device_ifs',
           'aim_device_cluster_devices',
           'aim_l3out_interfaces')
 
 
+def get_inspector():
+    """Reuse inspector"""
+    global _INSPECTOR
+
+    if _INSPECTOR:
+        return _INSPECTOR
+    else:
+        bind = op.get_bind()
+        _INSPECTOR = sa.engine.reflection.Inspector.from_engine(bind)
+    return _INSPECTOR
+
+
+def get_columns(table):
+    """Returns list of columns for given table."""
+    inspector = get_inspector()
+    return inspector.get_columns(table)
+
+
 def upgrade():
     for table in TABLES:
-        with op.batch_alter_table(table) as batch_op:
-            batch_op.alter_column(
-                'host', existing_type=sa.String(length=1024),
-                type_=sa.String(512))
+        columns = get_columns(table)
```

Comment:
do we really need to change this file though because the new migration script should be able to cover that?

Reply:
I verified with breakpoints that it will set it, regardless of the length of the string type above, so it is needed.